Sources/Sentry/SentrySessionTracker.m

```diff
@@ -32,46 +32,47 @@ - (instancetype)initWithOptions:(SentryOptions *)options
 
 - (void)start
 {
+
 #if SENTRY_HAS_UIKIT
-    NSNotificationName foregroundNotificationName = UIApplicationDidBecomeActiveNotification;
-    NSNotificationName backgroundNotificationName = UIApplicationWillResignActiveNotification;
-    NSNotificationName willTerminateNotification = UIApplicationWillTerminateNotification;
+    NSNotificationName didBecomeActiveNotificationName = UIApplicationDidBecomeActiveNotification;
+    NSNotificationName willResignActiveNotificationName = UIApplicationWillResignActiveNotification;
```

Comment:
L: more like a question, I remember we talked about this `willResign` and it was supposed to be gone, but it is here, is it not an issue anymore?

Reply:
I added a detailed comment on why. Let me know if it is clear for you now otherwise I change the comment.

Reply:
yep, crystal clear to me, thanks